net/databag/Makefile

```
@@ -0,0 +1,65 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=databag
+PKG_VERSION:=1.1.11
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
```

```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```

`v` is usually omitted in our usual practice, but it's okay to keep it as-is.